cognite_toolkit/_builtin_modules/contextualization/timeseries_equipment_sql/data_models/instance.Space.yaml

```diff
@@ -0,0 +1,4 @@
+space: '{{ instanceSpace }}'
+name: 'Instance space'
+description: |
+  This space contains data to contextualize.
```

Comment:
I don't think this space should be in this module. I expect it is already created.